packages/framework/src/Resources/styles/admin/libs/selectize/selectize-custom.less

```diff
@@ -19,24 +19,44 @@
 @selectize-color-dropdown-item-active: @color-message-info;
 @selectize-color-dropdown-item-active-text: @color-base;
 @selectize-color-highlight: @color-message-info;
+@selectize-padding-input: 6px;
+@selectize-right-padding-input: 25px;
 
 .selectize-control {
-    height: 32px;
-    line-height: 32px;
+    height: @input-height;
+    line-height: @input-height;
     max-width: 100%;
     padding: 0;
 
     border: none;
 
     box-shadow: none;
+
+    &.input {
+        width: auto;
+        min-width: @input-width;
+    }
 }
 
 .selectize-input {
-    padding: 6px;
+    height: 100%;
+    padding: @selectize-padding-input @selectize-right-padding-input @selectize-padding-input @selectize-padding-input;
+    overflow: hidden;
+
+    > * {
+        line-height: calc(~"@{input-height} - 2 * @{selectize-padding-input}");
+        height: calc(~"@{input-height} - 2 * @{selectize-padding-input}");
+        overflow: hidden;
+    }
+
+    input {
```

Comment:
is display:none necessary?

Reply:
Yes, it is. Selectize show after click input which we do not need it.